ios/RNSScreenStack.h

```diff
@@ -1,21 +1,37 @@
+#ifdef RN_FABRIC_ENABLED
+#import <React/RCTViewComponentView.h>
+#import <UIKit/UIKit.h>
+#else
 #import <React/RCTUIManagerObserverCoordinator.h>
 #import <React/RCTViewManager.h>
+#endif
 
 #import "RNSScreenContainer.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface RNScreensNavigationController : UINavigationController <RNScreensViewControllerDelegate>
 
 @end
 
-@interface RNSScreenStackView : UIView <RNSScreenContainerDelegate, RCTInvalidating>
+@interface RNSScreenStackView :
+#ifdef RN_FABRIC_ENABLED
+    RCTViewComponentView
+#else
+    UIView <RNSScreenContainerDelegate, RCTInvalidating>
+#endif
 
+#ifndef RN_FABRIC_ENABLED
 @property (nonatomic, copy) RCTDirectEventBlock onFinishTransitioning;
 
 - (void)markChildUpdated;
```

Comment:
Oh I think they belong to `RNScreensViewControllerDelegate` so technically they can be moved when `RNSScreenContainer` is added, but I would do it now anyway since they are empty in `ScreenStack`.